{{cookiecutter.project_slug}}/clients/web/react/tests/e2e/specs/test-login.cy.ts

```diff
@@ -2,12 +2,9 @@ describe('Tests login workflow', () => {
   it('Home page auto redirects to login', () => {
     cy.visit('/')
     cy.url().should('include', '/log-in')
-  })
-  it('Filling in email and passwords goes to', () => {
     cy.get('[data-cy=email]').type(Cypress.env('TEST_USER_EMAIL'))
     cy.get('[data-cy=password]').type(Cypress.env('TEST_USER_PASS'))
-    cy.get('[data-cy="login-btn"]').click()
-    cy.url().should('include', '/home')
+    cy.contains('[data-cy="login-btn"]', 'Login').click()
+    cy.url().should('include', '/test')
```

Comment:
To keep consistent with the vue side we should keep this as 2 separate tests.
The first test is nice as if it fails it usually means the app failed to build/load all together. But if it passes and the second one fails, it is usually a BE/data/config issue.

and of course...we need more/better tests here anyway. But that's a separate issue.

Reply:
Ok for simplicity can we approve this and I'll make the changes on a project I am working on. The reason I included it all here is because it fails as two separate ones, see this migration guide 
[here](https://docs.cypress.io/guides/references/migration-guide)